addons/advanced_ballistics/stringtable.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Which one is right?

Reply:
@MatsuoYutani What tool did you use to make these edits, and if it was bux's Tabler, which version?

Reply:
Hello @commy2  and @Blutze I used Bux's Tabler latest version 0.9.1

Reply:
@bux

Reply:
The new one has BOM, looks fine to me

Reply:
> The new one has BOM, looks fine to me.
> Indeed the ace one is missing the BOM

But we want without bom.

Reply:
Then the other way round :D
tabler checks wether the original file has BOM and only adds it in that case.
https://github.com/bux/tabler/commit/c67fd0dc69655878d30be6c0002f7ebdde928a82#diff-17e621989897af9f01c2714076d01920R154

Reply:
So... Do I have to remake it?

Reply:
On the "files changed" tab here on the PR, you can see in which files you caused changes to the very first line. If you go through them on your fork and open them the GitHub webclient's editor, you should be able to see a little red dot as the first character that you can remove like a normal one.